test/gtest/ucp/ucp_test.cc

```diff
@@ -250,9 +250,7 @@ void ucp_test::check_events(const std::vector<entity*> &entities,
         return;
     }
 
-    if ((req != NULL) &&
-        ((ucp_request_check_status(req) == UCS_ERR_UNREACHABLE) ||
-         (ucp_request_check_status(req) == UCS_ERR_REJECTED))) {
+    if ((req != NULL) && (ucp_request_check_status(req) == UCS_INPROGRESS)) {
```

Comment:
why is it needed? whoever calls this function already checks request status

Reply:
> why is if needed? whoever calls this function already checks request status

removed